src/components/DealProposalForm.tsx

```diff
@@ -151,9 +151,23 @@ export function DealProposalForm({
   onFileSelect: (file: File) => void;
   selectedFile: File | null;
 }) {
+  // Calculate total price
+  const startBlock = parseInt(dealProposal.startBlock) || 0;
+  const endBlock = parseInt(dealProposal.endBlock) || 0;
+  const pricePerBlock = parseInt(dealProposal.storagePricePerBlock) || 0;
```

Comment:
You don't have biome configured, otherwise it would have told you to use `Number.parseInt`